spec/basket_spec.rb

```diff
@@ -0,0 +1,37 @@
+describe 'Basket' do
+
+  let(:basket) {Basket.new}
+  let(:message_dbl) {double(:message, :send => "Success")}
+  let(:basket_dbl) {Basket.new(message_dbl)}
```

Comment:
I think the naming here is slightly confusing - at first I assumed that this was going to be a double of the basket class itself